.github/workflows/CI.yml

```diff
@@ -20,13 +21,19 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-18.04, ubuntu-16.04]
-        python-version: ['2.7']
+        python_version: ['2.7']
 
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 9999
 
+      - name: Install system packages
+        run: |
+          sudo apt-get update -qq || true
+          sudo apt-get install -y --no-install-recommends zlib1g-dev gcc g++ gfortran
+          sudo apt-get install -y mpi-default-bin mpi-default-dev build-essential
```

Comment:
Installing MPI triggers the scorep part of the code to get exercised. Scorep doesn't like MPICH though because the configure script wants to know which version of MPICH it is. Virtually all Linux distros default to OpenMPI as the default MPI implementation.